modules/text2vec-contextionary/helpers_test.go

```diff
@@ -222,7 +222,7 @@ func (fmp *fakeModulesProvider) ExtractAdditionalField(className, name string, p
 
 func (fmp *fakeModulesProvider) GetExploreAdditionalExtend(ctx context.Context, in []search.Result,
 	moduleParams map[string]interface{}, searchVector []float32,
-	argumentModuleParams map[string]interface{}, cfg moduletools.ClassConfig,
+	argumentModuleParams map[string]interface{}, cfg moduletools.ClassConfig, class *models.Class,
```

Comment:
do we need to pass a class here? module system is able to get the class from the search.Results

Reply:
There are situations where that isn't happening.  I'll take another look

Reply:
Removed